concepticondata/conceptlists/Lynott-2009-423.tsv

```diff
@@ -0,0 +1,424 @@
+Lynott-2009-423-255	255	orangey			1	4.23809523809524	1.57142857142857	0.380952380952381	3.76190476190476	4.33333333333333	1.13599128098599	1.69030850945703	1.07126982951031	1.44584791402007	1.06458129484475	Gustatory	0.276666666666667	0	0	7
+Lynott-2009-423-256	256	oval			1	4.61904761904762	3.95238095238095	0.047619047619048	0.19047619047619	0.285714285714286	0.804747816162957	0.864649667564296	0.218217890235992	0.679635756787974	0.783763812819726	Visual	0.50261780104712	7	2.07944154167984	4
+Lynott-2009-423-257	257	painful	PAINFUL	1129	1	3.85714285714286	4.61904761904762	3.33333333333333	0.761904761904762	0.80952380952381	1.45896244933563	0.973457265430305	1.62275485928508	1.4108423691101	1.40068010691405	Haptic	0.288256227758007	19	2.99573227355399	7
+Lynott-2009-423-258	258	palatable			0.80952380952381	3.47058823529412	0.764705882352941	0.235294117647059	3.64705882352941	4.82352941176471	1.17885787199006	0.970142500145332	0.562295714538387	1.27186754767292	0.39295262399669	Gustatory	0.354545454545454	1	0.693147180559945	9
+Lynott-2009-423-259	259	pale			1	4.95238095238095	0.285714285714286	0.238095238095238	0.095238095238095	0.333333333333333	0.218217890235994	0.643650304346789	0.624880941040924	0.436435780471985	1.06458129484475	Visual	0.82258064516129	35	3.58351893845611	4
+Lynott-2009-423-260	260	patterned			1	4.90476190476191	1.71428571428571	0.333333333333333	0.095238095238095	0.047619047619048	0.300792603759118	1.41925533794512	1.15470053837925	0.436435780471985	0.218217890235992	Visual	0.684563758389262	4	1.6094379124341	9
+Lynott-2009-423-261	261	peachy			1	3.23809523809524	2.14285714285714	0.047619047619048	3.04761904761905	3.61904761904762	1.70014005025356	1.93095091304037	0.218217890235992	1.59612626305661	1.53219419383414	Gustatory	0.295275590551181	0	0	6
+Lynott-2009-423-262	262	peppery	SPICY	544	1	2.66666666666667	0.333333333333333	0.095238095238095	3.52380952380952	4.95238095238095	1.35400640077266	0.658280588604383	0.436435780471985	1.24976188208185	0.218217890235994	Gustatory	0.419753086419753	0	0	7
+Lynott-2009-423-263	263	perfumed			1	0.476190476190476	0.19047619047619	0.095238095238095	4.90476190476191	2.28571428571429	0.928388260322567	0.601585207518238	0.436435780471985	0.436435780471984	1.45405835999994	Olfactory	0.604790419161677	1	0.693147180559945	8
+Lynott-2009-423-264	264	petite			1	4.52380952380952	2.71428571428571	0.285714285714286	0	0	1.12334534400814	1.67758329578168	0.902377811277357	0	0	Visual	0.60126582278481	2	1.09861228866811	6
+Lynott-2009-423-265	265	pink	PINK	3446	1	4.47619047619048	0.333333333333333	0.142857142857143	0.19047619047619	0.333333333333333	1.20909253653505	0.795822425754221	0.358568582800318	0.679635756787974	0.795822425754221	Visual	0.791304347826087	34	3.55534806148941	4
+Lynott-2009-423-266	266	plain	PLAIN	652	1	4.28571428571429	1.19047619047619	0.571428571428571	1.0952380952381	3.23809523809524	1.0555973258235	1.4
```

Comment:
The concept could use reworking, e.g. muscles can be SORE.

Reply:
Should we change it:

SORE: An injured, infected, inflamed or diseased muscle or patch of skin.

Reply:
Our SORE is a noun, so it shouldn't be mapped to this sore in any case.

Reply:
Yeah, something like that. The OED definition I think is nice but of course we just can't use it:

"Of wounds, hurts, ailments, or similar causes of physical suffering."

Reply:
But a new adjective sore would be a good idea, there is at least one mismapping on SORE.

Reply:
Yep